src/QualificationRunner.Core/Services/QualificationRunner.cs

```diff
@@ -295,31 +295,31 @@ private async Task createReportConfigurationPlan(QualificationRunResult[] runRes
 
       private JObject toJObject(object p) => _jsonSerializer.DeserializeFromString<dynamic>(_jsonSerializer.SerializeAsString(p));
 
-      private Task<QualificationRunResult> validateProject(QualifcationConfiguration qualificationConfiguration)
+      private Task<QualificationRunResult> validateProject(QualificationConfiguration configuration, ApplicationType application)

```

Comment:
thios type QualificationConfiguration and ApplicationType are now always together. Why don't we have ApplicationType a member of the QualificationConfiguration (that can be set automatically) instead of having so many instanced when we we move these two parameters togeher? (either as input or output? lots of cognitive overload for somethign that should be available readily

Reply:
Serialize where?

Reply:
Those `QualificationConfigurations` get serialized and the path is sent to the application for further processing.

Reply:
ok but they are created on line 322. even if the thing is serailized, who cares. Might actually be good to know where this is coming from?

Reply:
Yeah it's not a big deal if it's serialized. But for now the apps don't have any need to use this property